pkgs/unified_analytics/lib/src/enums.dart

```diff
@@ -70,21 +63,34 @@ enum DashEvent {
   });
 }
 
-/// Officially-supported clients of this package.
+/// Officially-supported clients of this package as logical
+/// tools, grouped by user point of view.
 ///
 /// All [label] values should use an underscore as a delimiter.
 enum DashTool {
-  dartTools(
-    label: 'dart_tools',
-    description: 'A CLI for Dart development',
+  dartTool(
+    label: 'dart-tool',
+    description: 'Dart CLI developer tool',
   ),
```

Comment:
Can we leave the previous value in too and add the new one. Then once this lands in the SDK, we can move over the code to the new enum value and then remove the old ones. That way it is not a breaking change, and we will not break the SDK build when DEPS is updated to pull in the new package.

Reply:
OK, I was thinking on the old model where DEPS were updated manually and I'd figure I'd land my changes on top of it, but I forgot there's now a semi-automatic update in place.   I can do this.